pkg/vm/memory/relocatable_test.go

```diff
@@ -43,6 +43,15 @@ func TestMaybeRelocatableAddFelt(t *testing.T) {
 	}
 }
 
+func TestRelocatableAddUint(t *testing.T) {
+	rel := memory.Relocatable{2, 4}
+	res := rel.AddUint(24)
+	expected := memory.Relocatable{2, 28}
+	if res != expected {
+		t.Fatalf("got wrong value from Relocatable.AddUint, expected: %v, got: %v", expected, res)
```

Comment:
```suggestion
		t.Errorf("got wrong value from Relocatable.AddUint, expected: %v, got: %v", expected, res)
```

Why have you decided to use Errorf?